client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1268,29 +1276,37 @@ public void setupMetaServiceRef(RpcEndpointRef endpointRef) {
   }
 
   private void slowStart() {
-    synchronized (currentMaxReqsInFlight) {
-      if (currentMaxReqsInFlight > maxInFlight) {
-        // Congestion avoidance
-        congestionAvoidanceFlag++;
-        if (congestionAvoidanceFlag >= currentMaxReqsInFlight) {
-          currentMaxReqsInFlight++;
-          congestionAvoidanceFlag = 0;
+    if (conf.pushDataSlowStart()) {
```

Comment:
I think changing currentMaxReqsInFlight to AtomicInteger is enough, no need to change here.

Reply:
As currentMaxReqsInFlight is already atomicInteger, maybe use atomic to update its value is better?

Reply:
done